chef/automate
# Proto changes to pass proxy connection_string to compliance service

Signed-off-by: Vivek Shankar <vshankar@progress.com>

### :nut_and_bolt: Description: What code changed, and why?


Since proxy configurations in inspec only work for cloud scan, we have removed the tests which earlier checked node scan.

### :chains: Related Resources

- https://github.com/chef/automate/issues/4958

### :+1: Definition of Done

- Automate should pass the proxy to ispec during scan
- Scans should pass

### :athletic_shoe: How to Build and Test the Change

```
rebuild components/automate-gateway
rebuild components/automate-cli
rebuild components/automate-deployment
```
- Setup a proxy
- Configure proxy in automate
```
[global.v1.proxy]
host = "<your proxy host>"
port = <your proxy port>
no_proxy = ["0.0.0.0", "127.0.0.1"]
```
- run AWS cloud scan
- Scan should pass

### :white_check_mark: Checklist

**All PRs** must tick these:


With occasional exceptions, all PRs **from Progress employees** must tick these:


All PRs **from Progress employees** should tick these if appropriate:


*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
